binds/python/bind_mutable.cpp

```diff
@@ -21,21 +21,34 @@ void bind_mutable_module(py::module& m) {
     using namespace py::literals;
 
 
-    py::class_<morphio::mut::Morphology>(m, "Morphology")
+    py::class_<morphio::mut::Morphology, std::shared_ptr<morphio::mut::Morphology>>(m, "Morphology")
         .def(py::init<>())
-        .def(py::init<const std::string&, unsigned int>(),
+        .def(py::init([](const std::string& filename, unsigned int options){
+            auto ptr = std::make_shared<morphio::mut::Morphology>();
+            ptr->init(filename, options);
+            return ptr;
+        }),
```

Comment:
Actually I was thinking about that yesterday and this will actually break inheritance, no? 
```
>>> class MyMorpho(Morphology):
>>>    def foo():
>>>          ...

>>> a = MyMorpho(file)
>>> type(a)
Morphology
>>> 
```

Reply:
Good question, you scared me but I just tried on ipython and it seems to work.
```
In [1]: class M(mio.mut.Morphology):
   ...:     pass
   ...:

In [2]: a = M('tests/data/simple.swc')

In [3]: a
Out[3]: <__main__.M at 0x10fc4c4d0>

In [4]: type(a)
Out[4]: __main__.M
```

Reply:
Cool I did not try yet !
I still need to understand exactly why but that s cool !